pages/docs/tools/asyncapi-cli.md

```diff
@@ -0,0 +1,137 @@
+---
+title: AsyncAPI CLI
+weight: 1
+---
+
+## Install CLI Using NPM and Node
+To use the AsynAPI CLI tool, you must have Node.js version 10 or higher and NPM installed on your PC. To check if you already have both installed, run the following commands in your terminal:
+
+```sh
+# check if node is installed
+node -v
+# or
+node --version
+
+# check if NPM is installed
+npm -v
+# or
+npm --version
+```
+
+If you don’t have Node.js or NPM installed, [install both with this package manager](https://nodejs.org/en/download/package-manager/).
+
+After installing Node.js and NPM, run the following command to install the AsyncAPI CLI globally on your PC:
```

Comment:
```suggestion
After installing Node.js and NPM, run the following command to install the AsyncAPI CLI globally:
```

You should not use the word PC in exchange for computer/machine/local environment. Not everyone has a PC, so it's incorrect to use that term.